### ISSUE TITLE
Compile issue on Ubuntu

### ISSUE THREAD

**Issue by PandaJim** [Faster3ck/Converseen#137]
The new release refused to build in all current Ubuntu releases, please help.

Error output:

```
[ 54%] Building CXX object CMakeFiles/converseen.dir/src/dialogconversionstatus.cpp.o
/usr/bin/c++ -DMAGICKCORE_HDRI_ENABLE=0 -DMAGICKCORE_QUANTUM_DEPTH=16 -DQT_CORE_LIB -DQT_GUI_LIB -DQT_NETWORK_LIB -DQT_NO_DEBUG -DQT_WIDGETS_LIB -I/<<PKGBUILDDIR>>/build -I/usr/include/ImageMagick-6 -I/usr/include/x86_64-linux-gnu/ImageMagick-6 -I/<<PKGBUILDDIR>>/. -isystem /usr/include/x86_64-linux-gnu/qt5 -isystem /usr/include/x86_64-linux-gnu/qt5/QtCore -isystem /usr/lib/x86_64-linux-gnu/qt5/mkspecs/linux-g++ -isystem /usr/include/x86_64-linux-gnu/qt5/QtGui -isystem /usr/include/x86_64-linux-gnu/qt5/QtNetwork -isystem /usr/include/x86_64-linux-gnu/qt5/QtWidgets -g -O2 -fdebug-prefix-map=/<<PKGBUILDDIR>>=. -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2   -Wall -W -std=c++11 -fPIC -fPIC -MD -MT CMakeFiles/converseen.dir/src/dialogconversionstatus.cpp.o -MF CMakeFiles/converseen.dir/src/dialogconversionstatus.cpp.o.d -o CMakeFiles/converseen.dir/src/dialogconversionstatus.cpp.o -c /<<PKGBUILDDIR>>/src/dialogconversionstatus.cpp
/<<PKGBUILDDIR>>/src/converter.cpp: In member function ‘Magick::Image Converter::convertPDFtoImage(Magick::Image&)’:
/<<PKGBUILDDIR>>/src/converter.cpp:339:12: error: ‘class Magick::Image’ has no member named ‘textAntiAlias’; did you mean ‘antiAlias’?
  339 |     ximage.textAntiAlias(true);
      |            ^~~~~~~~~~~~~
      |            antiAlias
make[3]: *** [CMakeFiles/converseen.dir/build.make:321: CMakeFiles/converseen.dir/src/converter.cpp.o] Error 1
make[3]: *** Waiting for unfinished jobs....
make[3]: Leaving directory '/<<PKGBUILDDIR>>/build'
make[2]: *** [CMakeFiles/Makefile2:86: CMakeFiles/converseen.dir/all] Error 2
make[2]: Leaving directory '/<<PKGBUILDDIR>>/build'
make[1]: *** [Makefile:139: all] Error 2
make[1]: Leaving directory '/<<PKGBUILDDIR>>/build'
dh_auto_build: error: cd build && make -j4 "INSTALL=install --strip-program=true" VERBOSE=1 returned exit code 2
make: *** [debian/rules:10: build] Error 25
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2
```

Full buildlog: https://launchpadlibrarian.net/682041393/buildlog_ubuntu-lunar-amd64.converseen_0.10.0.0-0build1~lunar_BUILDING.txt.gz

**Comment by Faster3ck** [comment]
Hi and thank you for reporting the problem. The error was caused by the fact that Ubuntu is still using ImageMagick 6 instead of version 7. Anyway, I have fixed the compile error in the new release 0.10.0.1: https://github.com/Faster3ck/Converseen/releases/tag/v0.10.0.1

Please let me know if the program compiles correctly!

**Comment by PandaJim** [comment]
Great, the new release works!
